internal/proposals/logs/dynamic-backend-configuration/overview.md

```diff
@@ -0,0 +1,149 @@
+
+```bash
+// operator
+kubectl create -f https://raw.githubusercontent.com/kubesphere/fluentbit-operator/master/manifests/setup/namespace-namespace.yaml
+kubectl create -f https://raw.githubusercontent.com/kubesphere/fluentbit-operator/master/manifests/setup/fluentbit-operator-crd.yaml
+kubectl create -f https://raw.githubusercontent.com/kubesphere/fluentbit-operator/master/manifests/setup/fluentbit-operator-clusterRole.yaml
+kubectl create -f https://raw.githubusercontent.com/kubesphere/fluentbit-operator/master/manifests/setup/fluentbit-operator-serviceAccount.yaml
+kubectl create -f https://raw.githubusercontent.com/kubesphere/fluentbit-operator/master/manifests/setup/fluentbit-operator-clusterRoleBinding.yaml
+kubectl create -f https://raw.githubusercontent.com/kubesphere/fluentbit-operator/master/manifests/setup/fluentbit-operator-deployment.yaml
+
+// fluent bit and fluent bit config
+kubectl create -f https://raw.githubusercontent.com/kubesphere/fluentbit-operator/master/manifests/quick-start/fluentbitconfig-fluentBitConfig.yaml
+kubectl create -f https://raw.githubusercontent.com/kubesphere/fluentbit-operator/master/manifests/quick-start/fluentbit-fluentBit.yaml
+
+// tail input
+kubectl create -f https://raw.githubusercontent.com/skhalash/community/logging-backend-dynamic-configuration/internal/proposals/logs/dynamic-backend-configuration/fluent-bit-operator/tail-input.yaml
+
+// stdout output
+kubectl create -f https://raw.githubusercontent.com/skhalash/community/logging-backend-dynamic-configuration/internal/proposals/logs/dynamic-backend-configuration/fluent-bit-operator/stdout-output.yaml
+```
+
+Inspect the logs of one of the Fluent Bit pods. It should print out raw logs collected by the container runtime, which look something like:
+```bash
+[17] kube.var.log.containers.fluent-bit-lcgh6_kubesphere-logging-system_fluent-bit-44cbdf54098e1cbb370215b18dca2a4f8f8660c08858756b4520d2c793c746a3.log: [1620154486.937270488, {"log"=>"{"log":"[3] kube.var.log.containers.ory-hydra-68f975fb7f-whfll_kyma-system_hydra-699cdccd54bf8589fc661d9011eb0f99f6fc27456e09b732efe333c4ba926cca.log: [1620154471.894581175, {\"log\"=\u003e\"{\"log\":\"time=\\\"2021-05-04T18:54:31Z\\\" level=info msg=\\\"completed handling request\\\" measure#hydra/admin: http://127.0.0.1:4444/.latency=160000 method=GET remote=\\\"127.0.0.1:59574\\\" request=/health/alive status=200 text_status=OK took=\\\"160µs\\\"\\n\",\"stream\":\"stderr\",\"time\":\"2021-05-04T18:54:31.563787749Z\"}\"}]\n","stream":"stdout","time":"2021-05-04T18:54:36.840279882Z"}"}]
+```
+
+Add a Kubernetes filter by executing the following command:
+```bash
+kubectl create -f https://raw.githubusercontent.com/skhalash/community/logging-backend-dynamic-configuration/internal/proposals/logs/dynamic-backend-configuration/fluent-bit-operator/kubernetes-filter.yaml
+```
+
+Inspect the logs of one of the Fluent Bit pods. Make sure that the logs are augmented with Kubernetes metadata.
```

Comment:
Is it easy to see that they contain Kubernetes metadata? I assume yes, but if it isn't, we should give an example what it looks like (maybe just a single small example "...metadata, such as `<typical-kubernetes-metadata-example>`.")

Reply:
It actually is quite trivial. I wanted to add a log entry example like the one above, but it was way too bulky ;)

Reply:
Do you think we should add it anyway or should we delete the example above for the sake of consistency?

Reply:
I'd say the first example is justified if that one isn't as trivial to recognise. If that one's also trivial, we can drop that too.
I'm usually obsessive about consistency but if you say it's bulky and doesn't add a lot of value, it's probably more user-friendly to favour brevity over consistency. If it were a screenshot instead of a code example, we'd also only add it if it adds helpful information.

Reply:
OK, cool, I'd also prefer not to add it then! Thanks ;)